### PR TITLE
test: matrix platform

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -17,12 +21,17 @@ jobs:
...
           npm install
           npm run package
-      - run: |
-          npx playwright test
+      - name: E2E test on win
+        run: npx playwright test
+        if: ${{ matrix.os != 'ubuntu-latest' }}
+        working-directory: ./src/test/e2e
+      - name: E2E test on linux
+        run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- npx playwright test
```
Comment (formulahendry) [inline .github/workflows/e2e.yml:29]: extra `--` before `npx`?

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -17,12 +21,17 @@ jobs:
         run: |
           npm install
           npm run package
-      - run: |
-          npx playwright test
+      - name: E2E test on win
+        run: npx playwright test
+        if: ${{ matrix.os != 'ubuntu-latest' }}
```
Comment (formulahendry) [inline .github/workflows/e2e.yml:26]: better to be `matrix.os == 'windows-latest'`?

Comment (wenytang-ms) [reply .github/workflows/e2e.yml:26]: maybe later we could support macos, but there still some problem, so not onboard the macos.
will investigate later

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (wenytang-ms) [reply .github/workflows/e2e.yml]: all can works, removed

**Review by formulahendry — APPROVED** [review]

**Pull request merged**